main.go

```diff
@@ -254,11 +255,30 @@ func InitFlags(fs *pflag.FlagSet) {
 		"Use deprecated infrastructure machine naming")
 	_ = fs.MarkDeprecated("use-deprecated-infra-machine-naming", "This flag will be removed in v1.9.")
 
+	fs.StringArrayVar(&syncMachineLabels, "additional-sync-machine-labels", []string{}, `The list of additional domains used to sync labels from machines to nodes. 
+The flag also supports standard glob processing:
```

Comment:
How sure are we that this definition from the filepath.Match godoc is identical to glob patterns?